uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/FloorFilterController.qml

```diff
@@ -48,9 +48,19 @@ QtObject {
 
     property ListModel levels: ListModel {}
 
-    property ListModel facilities: ListModel {}
+    property ListModel facilities: ListModel {
+        ListElement {
+            name: ""
+            modelId: ""
```

Comment:
ListModels needs to be initiated with roles, otherwise at line 289 in `floorfilter.qml`, a new `ListElement` is created and will  set the roles as no names. Meaning that when populating the `sites`, the roles set are null and no matching will be found, showing a delegate with no text. (was not expecting this from the docs: https://doc.qt.io/qt-5/qml-qtqml-models-delegatemodelgroup.html#insert-method-1
Not wokring Demo below:
https://user-images.githubusercontent.com/91129623/152151573-866a3437-481f-4dd3-a405-8b685f20c88a.mp4

Working demo below:
https://user-images.githubusercontent.com/91129623/152151941-8fc4416f-30c8-448c-b3f4-626326a3674c.mp4